CMSgov/design-system
# NO-TICKET: Expose 'informational' as part of the AlertVariation type

## Summary

When using Design System types, directly, in a project, the AlertVariation type is missing the string for the default Alert style, which is "informational," according to the docs. This update just adds that string to the union, but excludes it from the prop typing, since it's the default.

We are passing this detail via Adobe Analytics, so would like to have the "informational" typing exposed.

## How to test

There is no impact to the use of the component. When built, a project that imports `AlertVariation` from Design System types should see "informational" as one of the options (in our case, for analytics purposes).

## Checklist




### If this is a change to design:


### If this is a change to code:


### If this is a change to documentation:
